jekyll/_cci2/building-docker-images.md

```diff
@@ -14,20 +14,22 @@ This document explains how to build Docker images for deploying elsewhere or for
 
 ## Overview
 
-To build Docker images for deployment, you must use a special `setup_remote_docker` key which creates a separate environment for each build for security. This environment is remote, fully-isolated and has been configured to execute Docker commands. If your build requires `docker` or `docker-compose` commands, add the `setup_remote_docker` step into your `.circleci/config.yml`:
+To build Docker images for deployment, you must use a special `setup_docker_engine` key which creates a separate environment for each build for security. This environment is remote, fully-isolated and has been configured to execute Docker commands. If your build requires `docker` or `docker-compose` commands, add the `setup_docker_engine` step into your `.circleci/config.yml`:
 
 ```YAML
 jobs:
   build:
     steps:
       # ... steps for building/testing app ...
 
-      - setup_remote_docker
+      - setup_docker_engine
 ```
 
-When `setup_remote_docker` executes, a remote environment will be created, and your current [primary container][primary-container] will be configured to use it. Then, any docker-related commands you use will be safely executed in this new environment.
+When `setup_docker_engine` executes, a remote environment will be created, and your current [primary container][primary-container] will be configured to use it. Then, any docker-related commands you use will be safely executed in this new environment.
 
-*Note: `setup_remote_docker` is not curently compatible with the `machine` executor.*
+`setup_docker_engine` is now enabled automatically when initializing a machine executor. 
```

Comment:
I think I understand the idea behind this sentence but think it might be a bit confusing. Machine executor and `setup_remote_docker` are two separate options that enable you running docker command during the build. But they work differently and machine executor doesn't have limitations of remote docker. If we want to say what users can use machine executor when they need to run docker, I think, it might be better to just say it.

Reply:
Is the `setup_remote_docker` key supported with `machine`? Or should we keep this note about it not being compatible?

Reply:
It makes no sense to call setup_remote_docker from machine executor, so I would keep this comment but we can update it a bit to say that machine executor allows running docker commands by itself.

Reply:
ok, thanks, will do